project_euler/problem_54/test_poker_hand.py

```diff
@@ -0,0 +1,223 @@
+import os
+from itertools import chain
+from random import randrange, shuffle
+
+import pytest
+
+from .sol1 import PokerHand
+
+SORTED_HANDS = [
```

Comment:
If the order will not be changed at runtime, then please make this a tuple instead of a list.  That will slightly more efficient and will signal the reader that the order will not be modified at runtime.